apps/web/src/components/dialogs/delete-file-dialog.tsx

```
@@ -0,0 +1,78 @@
+
+			if (response.ok) {
+				onFileDeleted?.(file.id);
+				onOpenChange(false);
+
+				// Show success notification
+				const isFolder = file.mimeType === "application/vnd.google-apps.folder";
+				toast.success(`${isFolder ? "Folder" : "File"} ${file.name} deleted successfully`);
+			} else {
+				const error = await response.json();
+				console.error("Delete failed:", error);
+				toast.error("Failed to delete: " + (error.message || "Unknown error"));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)